evervault-inputs/src/main/java/com/evervault/sdk/input/ui/InlinePaymentCardInput.kt

```diff
@@ -12,10 +12,21 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
+import com.evervault.sdk.input.model.placeholder.PlaceholderDefaults
+import com.evervault.sdk.input.model.placeholder.PlaceholderTexts
+
+@Composable
+fun inlinePaymentCardInputLayout(): @Composable() (PaymentCardInputScope.(Modifier) -> Unit) =
+    { inline(modifier = it) }
```

Comment:
Moved to the top as it is the entry point (did not make it internal as it was exposed before)